pkg/apis/apis/v1alpha1/types.go

```diff
@@ -379,6 +414,47 @@ type APIExportPolicy struct {
 // of the API Export
 type LocalAPIExportPolicy struct{}
 
+const (
+	APIExportPermissionClaimLabelPrefix = "claimed.internal.apis.kcp.dev/"
+)
+
+type PermissionClaim struct {
+	GroupResource `json:","`
+
+	// This is the identity for a given APIExport that the APIResourceSchema belongs to.
+	// The hash can be found on APIExport and APIResourceSchema's status.
+	// It will be empty for core types.
+	// Note that one must look this up for a particular KCP instance.
+	// +optional
+	IdentityHash string `json:"identityHash,omitempty"`
+}
+
+func (p PermissionClaim) String() string {
+	if p.IdentityHash == "" {
+		return fmt.Sprintf("%v.%v", p.Group, p.Resource)
+	}
+	return fmt.Sprintf("%v.%v:%v", p.Group, p.Resource, p.IdentityHash)
```

Comment:
isn't it normally resource.group? And hence `resource:identity.group` here?

Reply:
updated